### ISSUE TITLE
gateway: Implement daemon for translating babeld to BGP Metrics

### ISSUE THREAD

**Issue by spolack** [freifunk-berlin/bbb-configs#989]
In order to attract traffic from internet on the closest gateway (mesh perspective), thus reducing load on gre tunnels we'll have to translate the mesh metric into a BGP metric. Only AS Path is usable for traffic engineering across multiple upstream autonomous systems.

This is already prepared on the bird side in https://github.com/freifunk-berlin/bbb-configs/blob/main/roles/cfg_openwrt/templates/gateway/bird.conf.j2#L14

Basically we'd just need the script to write the export policy in /dev/shm/announcement*.bird.conf and then trigger a bird reload.

The calculations could be done at a configurable interval.

I've written a very scary PoC to validate the strategy.

```
